.eslintignore

```diff
@@ -0,0 +1,19 @@
+# deps and build artifacts
+node_modules/
+dist/
+storybook-static/
+selection.json
+
+# dirs without any JS source
+src/base-styles/
+
+# ignored file types
```

Comment:
It might seem weird to ignore a bunch of file types instead of specifying `--ext` in the eslint config, but this is intentional.

When adding a new file type to `src`, it forces a decision on whether or not to ignore the extension or add the appropriate eslint plugin to handle the file.

The `ext` option is less likely to be updated when new file types are introduced to the linted dirs.